app/components/TuneCard/tests/index.test.js

```diff
@@ -0,0 +1,59 @@
+/**
+ *
+ * Tests for TuneCard
+ *
+ */
+
+import React from 'react';
+import { fireEvent } from '@testing-library/dom';
+import { renderWithIntl } from '@utils/testUtils';
+import TuneCard from '../index';
+
+describe('<TuneCard />', () => {
+  it('should render and match the snapshot', () => {
+    const artistName = 'Alan Walker';
+    const collectionName = 'Some Collection';
+    const { baseElement } = renderWithIntl(<TuneCard artistName={artistName} collectionName={collectionName} />);
+    expect(baseElement).toMatchSnapshot();
+  });
+
+  it('should call handlePlay on Click', () => {
+    const { getByTestId } = renderWithIntl(<TuneCard />);
+    const handlePlay = jest.fn();
+    fireEvent.click(getByTestId('play-pause-btn'), { onclick: handlePlay() });
+    expect(handlePlay).toHaveBeenCalled();
+  });
+
+  it('should play/pause audio when button is clicked', () => {
+    const playStub = jest.spyOn(window.HTMLMediaElement.prototype, 'play').mockImplementation(() => {});
+    const pauseStub = jest.spyOn(window.HTMLMediaElement.prototype, 'pause').mockImplementation(() => {});
+
+    const { getByTestId } = renderWithIntl(<TuneCard />);
+
+    const button = getByTestId('play-pause-btn');
+
+    fireEvent.click(button);
```

Comment:
just so its amply clear can we do this 
```
expect(playStub).not.toHaveBeenCalled();
fireEvent.click(button);
expect(playStub).toHaveBeenCalled();

expect(playStub).not.toHaveBeenCalled();
fireEvent.click(button);
expect(pauseStub).toHaveBeenCalled();
```

Reply:
Removed the test as the implementation has changed in next commit